{{ cookiecutter.project_slug }}/setup.py

```diff
@@ -33,14 +66,14 @@ def get_version(*file_paths):
     author='eduNEXT',
     author_email='contact@edunext.co',
     packages=[
-        '{{ cookiecutter.package_name }}'
+        '{{ cookiecutter.package_name }}',
     ],
     include_package_data=True,
-    install_requires=[],
+    install_requires=load_requirements('requirements/base.in'),
```

Comment:
Often we put requirements in base that are just for testing or running django commands without the devstack.
Should we use a special file for the ones that go in the edxapp venv?

Reply:
No @felipemontoya according to this proposal: https://open-edx-proposals.readthedocs.io/en/latest/oep-0018-bp-python-dependencies.html We should follow those guidelines.